portality/templates/doaj/article.html

```diff
@@ -61,16 +63,21 @@
     <div class="span4">
         <div class="box with-borders" style="height: 120px">
             {% if article.bibjson().journal_title %}
+                {% if article.bibjson().get_identifiers(idtype='pissn') or article.bibjson().get_identifiers(idtype='eissn') %}
+                    <p><strong>Journal</strong>:&nbsp;
+                    <a href={{ "/toc/" + article.bibjson().get_identifiers(idtype='pissn')[0] or article.bibjson().get_identifiers(idtype='eissn')[0]}}>{{ article.bibjson().journal_title }}</a></p>
```

Comment:
Wait what :D? So this will output /toc/pissn if a pissn is present, or if it isn't, it'll do /toc/eissn? Never seen an or in a jinja expression evaluation (rather than control flow block, with {% %}).